self-assessment.md

```diff
@@ -79,21 +117,29 @@ This document provides the CNCF TAG-Security with an initial understanding of k8
 
 ## Security functions and features
 
+@ytsarev please check this section.
+
 | Component | Applicability | Description of Importance |
 | --------- | ------------- | ------------------------- |
 | Component Name 1 | `Critical` or `Security Relevant` | why this feature is an important part of k8gb’s design and why it should be part of the threat model |
```

Comment:
@elohmrow, do you know where I can read the definitions of `Critical` and `Security Relevant`?

Reply:
@ytsarev https://github.com/cncf/tag-security/blob/34a758dbd13259cb0b89c626b483caac0cda0449/community/assessments/guide/self-assessment.md#security-functions-and-features